static/app/components/profiling/profilingOnboardingSidebar.tsx

```diff
@@ -340,6 +340,9 @@ const Introduction = styled('div')`
 
 const Content = styled('div')`
   padding: ${space(2)};
+  display: flex;
+  flex-direction: column;
+  gap: ${space(1)};
```

Comment:
adding this minor fix here too.

**Before**
<img width="410" alt="image" src="https://github.com/user-attachments/assets/d04db72b-894e-4e37-ad05-b9022d05f8b3">


**After**
<img width="281" alt="image" src="https://github.com/user-attachments/assets/5b9f3424-3487-4061-b689-2671dd581619">